persistence/src/commonTest/kotlin/com/wire/kalium/persistence/utils/stubs/MessageStubs.kt

```diff
@@ -93,3 +96,243 @@ fun newDraftMessageEntity(
+            id = "testMessage18",
+            content = MessageEntityContent.ConversationProtocolChanged(ConversationEntity.Protocol.MIXED)
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage19",
+            content = MessageEntityContent.ConversationProtocolChangedDuringACall
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage20",
+            content = MessageEntityContent.HistoryLostProtocolChanged
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage21",
+            content = MessageEntityContent.HistoryLost
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage22",
+            content = MessageEntityContent.ConversationCreated
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage23",
+            content = MessageEntityContent.ConversationDegradedMLS
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage24",
+            content = MessageEntityContent.ConversationVerifiedMLS
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage25",
+            content = MessageEntityContent.ConversationDegradedProteus
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage26",
+            content = MessageEntityContent.ConversationVerifiedProteus
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage27",
+            content = MessageEntityContent.ConversationStartedUnverifiedWarning
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage28",
+            content = MessageEntityContent.Federation(
+                listOf("otherDomain"),
+                MessageEntity.FederationType.DELETE
+            )
+        ),
+        newSystemMessageEntity(
+            conversationId = conversationId,
+            senderUserId = senderUserId,
+            id = "testMessage29",
+            content = MessageEntityContent.LegalHold(
+                listOf(QualifiedIDEntity("otherId", "otherDomain")), MessageEntity.LegalHoldType.ENABLED_FOR_MEMBERS
+            )
+        ),
+    )
+}
```

Comment:
can we do the same test but when updating the conversaion ID it self 
add messages -> run query to update the conversation ID (this is the case for group conversations)